### PR TITLE
Add screen prompts for debug mode

### PR DESCRIPTION
**Pull request by MegaPiggy**

### CODE REVIEW

File: NewHorizons/Utility/ImageUtilities.cs [hunk 1]
```diff
@@ -351,7 +351,16 @@ public static Texture2D MakeSolidColorTexture(int width, int height, Color color
             newTexture.Apply();
             return newTexture;
         }
-        
+
+        public static Sprite GetButtonSprite(JoystickButton button) => GetButtonSprite(ButtonPromptLibrary.SharedInstance.GetButtonTexture(button));
+        public static Sprite GetButtonSprite(KeyCode key) => GetButtonSprite(ButtonPromptLibrary.SharedInstance.GetButtonTexture(key));
+        private static Sprite GetButtonSprite(Texture2D texture)
+        {
+            var sprite = Sprite.Create(texture, new Rect(0, 0, texture.width, texture.height), new Vector2(0.5f, 0.5f), 100, 0, SpriteMeshType.FullRect, Vector4.zero, false);
```
Comment (JohnCorby) [inline NewHorizons/Utility/ImageUtilities.cs:359]: will this do a funny memory leak?

Comment (MegaPiggy) [reply NewHorizons/Utility/ImageUtilities.cs:359]: I don't think creating a sprite will cause one. There is a few other things that create sprites (like vessel coordinate prompt, ship log star chart, ship log rumor mode, and ship log map mode.

File: NewHorizons/Utility/DebugUtilities/DebugPropPlacer.cs [hunk 2]
```diff
@@ -39,14 +40,36 @@ private struct PropPlacementData
...
+        private ScreenPrompt _redoPrompt;
+
         private void Awake()
         {
             _rc = this.GetRequiredComponent<DebugRaycaster>();
             currentObject = DEFAULT_OBJECT;
+
+            _placePrompt = new ScreenPrompt(TranslationHandler.GetTranslation("DEBUG_PLACE", TranslationHandler.TextType.UI) + " <CMD>", ImageUtilities.GetButtonSprite(KeyCode.G));
+            _undoPrompt = new ScreenPrompt(TranslationHandler.GetTranslation("DEBUG_UNDO", TranslationHandler.TextType.UI) + " <CMD>", ImageUtilities.GetButtonSprite(KeyCode.Minus));
+            _redoPrompt = new ScreenPrompt(TranslationHandler.GetTranslation("DEBUG_REDO", TranslationHandler.TextType.UI) + " <CMD>", ImageUtilities.GetButtonSprite(KeyCode.Equals));
```
Comment (xen-42) [inline NewHorizons/Utility/DebugUtilities/DebugPropPlacer.cs:54]: This button is blank

**Review by xen-42 — CHANGES_REQUESTED** [review]

File: NewHorizons/Utility/DebugUtilities/DebugPropPlacer.cs (hunk 2 above)
Comment (MegaPiggy) [reply NewHorizons/Utility/DebugUtilities/DebugPropPlacer.cs:54]: Yeah, I know. I was trying to get that bug fixed in Vanilla Fix. https://github.com/JohnCorby/ow-vanilla-fix/pull/5

**Review by xen-42 — APPROVED** [review]

**Pull request merged**